src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/OssArtifactVersionScore.java

```diff
@@ -0,0 +1,104 @@
+package com.sap.oss.phosphor.fosstars.model.score.oss;
+
+import com.sap.oss.phosphor.fosstars.model.Score;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.score.WeightedCompositeScore;
+import com.sap.oss.phosphor.fosstars.model.value.ScoreValue;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.oss.phosphor.fosstars.model.weight.ImmutableWeight;
+import com.sap.oss.phosphor.fosstars.model.weight.ScoreWeights;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * <p>The artifact version score evaluates how the given version relates to
+ * released versions, the release history of artifacts and if there are
+ * known vulnerabilities for the given artifact version
+ * of an open-source project.
+ * The score is based on the following sub-scores:</p>
+ * <ul>
+ *   <li>{@link ArtifactVersionScore}</li>
+ *   <li>{@link ArtifactReleaseHistoryScore}</li>
+ *   <li>{@link ProjectActivityScore}</li>
+ *   <li>{@link ArtifactVersionVulnerabilityScore}</li>
+ * </ul>
+ */
+public class OssArtifactVersionScore extends WeightedCompositeScore {
```

Comment:
```suggestion
public class OssArtifactVersionSecurityScore extends WeightedCompositeScore {
```

Reply:
But we already have `OssArtifactSecurityScore`. I think we need to come up with better and more specific score names.

Reply:
I am not sure I understand why this scoring function includes `ArtifactVersionVulnerabilityScore`. Should it be moved to `OssArtifactSecurityScore`? Then, this scoring function can check age/history/etc -- maybe then it is going to be easier to tell what it assesses.

Reply:
Or, it may be even better to merge this score to `OssArtifactSecurityScore`.